src/features/presentation_mode/CurrentEvent/CurrentEvent.tsx

```
@@ -0,0 +1,24 @@
+export const CurrentEvent = () => {
+  const [{ data: currentEvent }] = useAtom(currentEventResourceAtom.v3atom);
+
+  const layoutContextValue = useUniLayoutContextValue({
+    layout: eventCardLayoutTemplate,
+  });
+
+  if (!currentEvent) return null;
+
+  return (
+    <div className="knt-panel knt-current-event" style={{ zoom: 1.4 }}>
+      <UniLayout data={currentEvent} layout={eventCardLayoutTemplate}></UniLayout>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)